exercises/simple-linked-list/example.js

```diff
@@ -0,0 +1,119 @@
+function Element (value, next) {
+  if (!(this instanceof Element)) {
+    throw new Error('Element is a constructor.');
+  }
+
+  if (value === undefined) {
+    throw new Error('Value required.');
+  }
+
+  if (next !== undefined && !(next instanceof Element)) {
+    throw new Error('A Element instance as next value is required.');
+  }
+
+  this.value = value;
+  this.next = next;
+}
+
+function List () {}
+
+List.prototype.push = function (value) {
+  if (value === undefined) {
+    throw new Error('Argument required.');
+  }
+
+  var newEl = (value instanceof Element)
+    ? value
+    : new Element(value);
+
+  if (!this.head) {
+    this.head = newEl;
+    return;
+  }
+
+  var lastEl = this.head;
+  while (lastEl.next) {
+    lastEl = lastEl.next;
+  }
+
+  lastEl.next = newEl;
+}
```

Comment:
I hate to even mention this on your beautiful pull request, but all these function expressions should have terminating semicolons...this is why we need a linter!  Would you mind adding them in the meantime?